isign/archive.py

```diff
@@ -189,7 +189,7 @@ def precheck(cls, path):
                 if plist_path not in zipfile_obj.namelist():
                     return False
                 plist = cls.get_info(relative_bundle_dir, zipfile_obj)
-        is_native = IosApp.is_native(plist)
```

Comment:
this entire precheck is dubious, it only delays the failure

I think I wrote this b/c we had so many other other uploads to sift through at that other company